location.md

```diff
@@ -4,17 +4,61 @@ title: "All roads lead to Rome"
 permalink: /location/
 ---
 
-<div class="backdrop">
+<div class="backdrop" style="background: url(/assets/rome/the-city.jpg) 50% 40%; background-repeat:no-repeat; padding: 0; margin-bottom: 1em;" >
   <div class="popout">
     <section>
       <h1>Welcome to {{site.location.city}}</h1>
-      <p>We are very pleased to host the fourth edition of RustFest in Rome, Italy.</p>
+      <p> We are very pleased to host the fourth edition of RustFest in Rome, Italy. </p>
     </section>
   </div>
 </div>
 
-<section>
-<p>
-Rome is the capital of Italy.
-</p>
+<section markdown="1">
 
+Rome, the eternal city, is the capital of Italy with 3 million of residents. It's located in the central-western portion of the Italian Peninsula, within Lazio, along the shores of the Tiber. 
+Rome's history spans 28 centuries, leaving behind a huge number of monuments spanning from the famous Colosseum, to Piazza Di Spagna and Fontana Di Trevi.
+Also the Vatican City is an independent country inside the city boundaries of Rome, the only existing example of a country within a city: for this reason Rome has been often defined as capital of two states.  
```

Comment:
Leave out the "also".
Replace the : with a . and make the next its own  sentence